src/Uno.Foundation/DelegateCommand.T.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows.Input;
+
+namespace Uno.UI.Common
+{
+	public class DelegateCommand<T> : ICommand
+	{
+		private Action<T> _action;
+
+		public event EventHandler CanExecuteChanged;
+
+		public DelegateCommand(Action<T> action)
+		{
+			_action = action;
+			OnCanExecuteChanged(true);
```

Comment:
CanExecuteChanged cannot have been registered at this point.

Reply:
Copy/paste from DelegateCommand, I'll fix both